### ISSUE TITLE
Old color scheme?

### ISSUE THREAD

**Issue by sobolevn** [ayu-theme/vscode-ayu#123]
Hi! Thanks a lot for an awesome theme. I am using it for like 4 years now.
Dotfiles: https://github.com/sobolevn/dotfiles#vs-code

My main question is: after a recent update (1-2 days ago) my color scheme has change a lot. And I don't like it, sorry 🙂

Let's discuss new problems I see:
<img width="1280" alt="Снимок экрана 2021-09-03 в 22 06 44" src="https://user-images.githubusercontent.com/4660275/132054919-67011b0f-814f-47df-8998-b118d305274f.png">
From top to bottom:
- [ ] New green highlight of `()`, which was not used before. It now has too many "attention" attracted to it
- [ ] Purple function arguments, it just does not fit the color scheme
- [ ] New color of message panel on hover is too light

Related https://github.com/dempfi/ayu/issues/250 

Is there something you can suggest here?
Or should I just fork it?

**Comment by dempfi** [comment]
4 years is really a lot of time. I'm happy to hear people loving and using ayu ❤️

Thank for detailed report, it seems all of this can be addressed:
- I'll look into toning down a notch bracket highlight. I can see how that can be annoying;
- Purple was used for arguments for a couple years now. What has changed tho is that I enabled semantic highlighting which causes this arguments color to be used in places of usage too. For now, I can recommend to disable semantic highlighting
- Notifications color prominence is linked to list selection color and since I aimed to make list selections brighter and more readable, that also got applied to notifications; I'll check if it's possible to reduce contrast of those notifications;

**Comment by dempfi** [comment]
@emlautarom1 when you say 
> Text used to be yellow, and now is pink-ish.

Do you mean the `false` constants or something else?

**Comment by dempfi** [comment]
I've toned down bracket highlights.

**Comment by xuhaibahmad** [comment]
TYSM for this theme. I use Ayu Dark for pretty much every IDE I use.

Please reconsider these recent changes, especially the purple accent. I haven't tried other languages with them yet, but currently, my Dart code looks like this (v1.0.2):

![image](https://user-images.githubusercontent.com/6507679/132085588-efca0aa6-9572-4765-b8a3-b193fe2f2994.png)

**Comment by dempfi** [comment]
Wow, that's really terrible @xuhaibahmad. Can you share a piece of code for me to test on?

**Comment by xuhaibahmad** [comment]
Sure! I think the semantic highlighting will have this issue with languages/frameworks that uses object nesting such as Flutter, SwiftUI, and JetPack Compose, etc.

``` dart
import 'package:flutter/foundation.dart';
import 'package:flutter/material.dart';

class ClickableMaskView extends StatelessWidget {
  const ClickableMaskView({
    Key? key,
    required this.rippleColor,
    required this.onClick,
    required this.padding,
    required this.radius,
  }) : super(key: key);

  final Color rippleColor;
  final Function onClick;
  final double padding;
  final double radius;

  @override
  Widget build(BuildContext context) {
    return Positioned.fill(
      child: Container(
        padding: EdgeInsets.symmetric(horizontal: padding),
        child: Material(
          color: rippleColor,
          child: InkWell(
            onTap: () => onClick.call(),
            borderRadius: BorderRadius.circular(radius),
          ),
        ),
      ),
    );
  }
}

```

**Comment by sobolevn** [comment]
The same with python named arguments:
<img width="1280" alt="Снимок экрана 2021-09-04 в 11 24 13" src="https://user-images.githubusercontent.com/4660275/132088118-1e269f0b-0d2f-46f5-a9c5-9b84d682e364.png">


```python
setup(
    name="django-stubs",
    version="1.8.0",
    description="Mypy stubs for Django",
    long_description=readme,
    long_description_content_type="text/markdown",
    license="MIT",
    url="https://github.com/typeddjango/django-stubs",
    py_modules=[],
    python_requires=">=3.6",
    install_requires=dependencies,
)
```

**Comment by sobolevn** [comment]
One more problem: new search is very low in contrast. It is hard to see the highlighted text.
<img width="273" alt="Снимок экрана 2021-09-04 в 12 42 32" src="https://user-images.githubusercontent.com/4660275/132090133-05df10ca-1348-4bc0-98c8-883799a09e3b.png">

I really hope that it will be possible to restore things back to normal soon 🙂

**Comment by dempfi** [comment]
Can you guys give a go to this version [ayu-1.0.2.vsix.zip](https://github.com/ayu-theme/vscode-ayu/files/7110357/ayu-1.0.2.vsix.zip)

**Comment by sobolevn** [comment]
@dempfi thanks a lot! Brackets and keyword arguments are back to normal! 👍 
However, function arguments, search results, and constants are still purple for some reason 😞 

<img width="1280" alt="Снимок экрана 2021-09-04 в 22 19 04" src="https://user-images.githubusercontent.com/4660275/132105718-aa93a870-c8b8-4746-9791-24c435e3a537.png">

**Comment by xuhaibahmad** [comment]
Much better! Though the use of purple for parameters means a lot of my code is still that color. Also, noticed that parameters are not colored when I disable semantic highlighting.

Semantic Highlighting ON:

![image](https://user-images.githubusercontent.com/6507679/132105875-a6e9f9a0-896f-45f7-81dd-b45307820714.png)

Semantic Highlighting OFF:

![image](https://user-images.githubusercontent.com/6507679/132105898-ec094537-5d46-4083-851f-cade1e955c40.png)

**Comment by sobolevn** [comment]
Ok, I found the solution for my problem. Sharing with other people:

You can use older versions of extensions in vscode. 
https://stackoverflow.com/questions/42626065/vs-code-rollback-extension-install-specific-extension-version/53755378

Demo (and the final result, code color is back to normal):

https://user-images.githubusercontent.com/4660275/132106037-b87ed8db-7e96-4e98-821c-11ff619e69f2.mov

**Comment by naivelogic** [comment]
i had to rollback to v0.20.2. love ayu tho!!

**Comment by exitlol** [comment]
Hope I am not late for the party.
@dempfi I work with Angular/TS. And the yellow accent colors are still purple, the type associations are the same color as the Interface

screens from v.1.0.0 and up:
![image](https://user-images.githubusercontent.com/36930188/132172972-b619c9be-f91d-4d7c-b483-6cb90ced2dde.png)
![image](https://user-images.githubusercontent.com/36930188/132172999-2e3db557-c864-4e4a-8194-8a9048ca35c7.png)
I think this is the worst part:
![image](https://user-images.githubusercontent.com/36930188/132173364-11f5d2b7-b832-4021-b892-8e19c8bb912c.png)

screens from same code before v.1.0.0:
![image](https://user-images.githubusercontent.com/36930188/132173517-15e9b6fc-cc1a-4a11-bf2b-98e170367174.png)
![image](https://user-images.githubusercontent.com/36930188/132173547-a3c26b17-971b-4326-8880-faf257364d25.png)
![image](https://user-images.githubusercontent.com/36930188/132173471-7d43d663-459a-4f67-bd0b-e9f8da813b7a.png)

If you need snippets or anything else let me know.

**Comment by emlautarom1** [comment]
> @emlautarom1 when you say
> 
> > Text used to be yellow, and now is pink-ish.
> 
> Do you mean the `false` constants or something else?

Yes, exactly that. Notice that the constant `2` is also pink-ish

**Comment by exitlol** [comment]
Any updates in this issue?
